specs/experimental/op-contracts-manager.md

```diff
@@ -20,146 +20,247 @@ of governance approved [contract releases] can be found on the
+/// format `op-contracts/vX.Y.Z`.
+function latestRelease() external view returns (string memory);
 
-The privileged methods of the OP Contracts Manager will be held by the L1 ProxyAdmin owner, as
-specified by the [standard configuration].
+    /// @notice Represents the interface version so consumers know how to decode the DeployOutput struct
+    function OUTPUT_VERSION() external view returns (uint256);
+    /// @notice Addresses of the Blueprint contracts.
+    function blueprints() external view returns (Blueprints memory);
+    /// @notice Maps an L2 chain ID to an L1 batch inbox address
+    function chainIdToBatchInboxAddress(uint256 _l2ChainId) external pure returns (address);
+    /// @notice Addresses of the latest implementation contracts.
+    function implementations() external view returns (Implementations memory);
+    /// @notice L1 smart contracts release deployed by this version of OPCM.
+    function l1ContractsRelease() external view returns (string memory);
+    /// @notice Address of the ProtocolVersions contract shared by all chains.
+    function protocolVersions() external view returns (address);
+    /// @notice Address of the SuperchainConfig contract shared by all chains.
+    function superchainConfig() external view returns (address);
+    /// @notice Maps an L2 Chain ID to the SystemConfig for that chain.
+    function systemConfigs(uint256) external view returns (address);
+    /// @notice Semver version specific to the OPContractsManager
+    function version() external view returns (string memory);
+
+/// @notice Returns the implementation contract addresses.
+function implementations() public view returns (Implementations memory);
 
-### `deploy`
+/// @notice Maps an L2 Chain ID to the SystemConfig address for that chain.
+/// @notice All contracts for a chain can be found from its SystemConfig.
+function systemConfigs(uint256 chainId) external view returns (SystemConfig);
 
-The `deploy` method is the only non-view method in the contract. It is used to
-deploy the full set of L1 contracts required to setup a new OP Stack chain that
-complies with the [standard configuration]. It has the following interface:
+/// @notice Maps an L2 chain ID to an L1 batch inbox address as defined by the standard
+function chainIdToBatchInboxAddress(uint256 _l2ChainId) public pure returns (address);
 
-```solidity
-struct Roles {
-  address proxyAdminOwner;
-  address systemConfigOwner;
-  address batcher;
-  address unsafeBlockSigner;
-  address proposer;
-  address challenger;
-}
+/// @notice Returns the blueprint contract addresses.
+function blueprints() public view returns (Blueprints memory);
+```
+
+## Deployment
+
+### Interface
 
-function deploy(
-  uint256 l2ChainId,
-  uint32 basefeeScalar,
-  uint32 blobBasefeeScalar,
-  Roles roles
-) external returns (SystemConfig)
+Version 1.0.0 of the OP Contracts Manager deploys the `op-contracts/v1.6.0` contracts release,
```

Comment:
This is going to fall out of date pretty quickly, "in the future this will...". Perhaps we want a table of opcm version -> contracts release

Reply:
This statement isn't new, just relocated from elsewhere in the doc.

I've removed it, as it feels like user facing documentation, not instructions for implementation.

60ecdec71700fbbbe7605edd616b16cd24753f6e